buildkite/scripts/connect-to-testnet-on-develop.sh

```diff
@@ -28,19 +28,7 @@ apt-get install --allow-downgrades -y curl ${PROJECT}=${VERSION}
 TESTNET_NAME="turbo-pickles"
 
 
-if [ ! -d coda-automation ]; then
-  # Somebody ran this without the mina repo checked out...
-  echo "WARNING: Connecting to testnet without a checked-out coda-automation repo. Attempting to pull data from github's master branch (fallback branch is 3a4e5ce2d)."
-  mkdir -p coda-automation/terraform/testnets/$TESTNET_NAME
-  # Fetch the files we need from coda-automation's master instead
-  # Fall through to a known-good file
-  curl https://raw.githubusercontent.com/MinaProtocol/coda-automation/master/terraform/testnets/$TESTNET_NAME/genesis_ledger.json --output coda-automation/terraform/testnets/$TESTNET_NAME/genesis_ledger.json \
-  || curl https://raw.githubusercontent.com/MinaProtocol/coda-automation/3a4e5ce2dc1ff01dde37495d43979aa1aeb20bb5/terraform/testnets/$TESTNET_NAME/genesis_ledger.json  --output coda-automation/terraform/testnets/$TESTNET_NAME/genesis_ledger.json
-  curl https://raw.githubusercontent.com/MinaProtocol/coda-automation/master/terraform/testnets/$TESTNET_NAME/peers.txt --output coda-automation/terraform/testnets/$TESTNET_NAME/peers.txt \
-  || curl https://raw.githubusercontent.com/MinaProtocol/coda-automation/3a4e5ce2dc1ff01dde37495d43979aa1aeb20bb5/terraform/testnets/$TESTNET_NAME/peers.txt  --output coda-automation/terraform/testnets/$TESTNET_NAME/peers.txt
-else
-  cd coda-automation && git pull origin master && cd -
-fi
+cd coda-automation && git pull origin master && cd -
```

Comment:
we shouldn't check in the git repo for coda-automation right? I don't think you need this if master is all merged in. We would just delete coda-automation git altogether right?

Reply:
Yep good catch, thanks. I'll just remove that line. I think we shouldn't delete the whole repo quite yet because we still use peers.txt from coda-automation in the docs. After the docs are updated we can though I think